plugins/infinispan90/src/main/java/org/radargun/service/Infinispan90Lifecycle.java

```diff
@@ -0,0 +1,24 @@
+package org.radargun.service;
+
+import org.infinispan.remoting.transport.jgroups.JGroupsTransport;
+import org.jgroups.JChannel;
+
+/**
+ * @author Roman Macor (rmacor@redhat.com)
+ */
+public class Infinispan90Lifecycle extends Infinispan52Lifecycle {
+
+   public Infinispan90Lifecycle(Infinispan90EmbeddedService service) {
+      super(service);
+   }
+
+   @Override
+   protected JGroupsTransport getTransport() {
```

Comment:
The code is identical to 5.2; I recall needing to do similar things, though - what was your error? Adding a comment explaining why this is necessary could clarify... And no need for parentheses here, btw.